pkgs/by-name/ro/roon-tui/package.nix

```
@@ -0,0 +1,32 @@
+{ stdenv
+, lib
+, rustPlatform
+, fetchFromGitHub
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "roon-tui";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "TheAppgineer";
+    repo = pname;
```

```suggestion
    repo = "roon-tui";
```

https://github.com/NixOS/nixpkgs/issues/277994